### ISSUE TITLE
Data Problems F-16-C

### ISSUE THREAD

**Issue by Greenislands** [dcs-bios/dcs-bios#247]
Hi,

I am using the F-16C module. Probably there was an update there which causes some issues in the CommonData read values of:

Indicated Airspeed KM H
                _onIasEuChange_                        value = jÂ                  (ingame: 13 kmh)

Altitude MSL (ft)
                _onAltMslFtChange_                   value = 48565           (ingame: 4.410 ft)

Indicated airspeed values are not numerical numbers. It seems there is the whole asci table shown when values are changing.

In the F-16C_50 module, folder UHF:

UHF Manual Frequency Display
      _onUhfFreqDispChange_             value = none at all  (ingame: the frequency)

Thank you for reading

Versions:
dcs-bios v.0.10.0+64
module-f-16-c-50 7e9f03bbb85a4c434a8f1d79678089657e96cbd0 (latest)
dcs beta v 2.5.6.59625

**Comment by Slamraam** [comment]
Yes I can confirm that the UHF freq string for the F16C never shows a thing. But my solution is to use the separate frequency knob code snippets with the unsigned integer phrases, you can also print that value on your screen. The 0.25 kHz value you need to adjust that by multiplying the integer value *25, because otherwise you only get the switch position which is 0,1,2,3. Hope this helps.
